src/YahooFinance.py

```diff
@@ -198,9 +198,44 @@ def parse_modules(self, content):
     results = data.get('quoteSummary', {}).get('result', None)
     if not results:
       logging.error('Could not parse response for url: ' + self.url)
-      return
+      return False
     for module in self.modules:
       for result in results:
         if module in result:
           self.module_data[module] = result[module]
-          break
\ No newline at end of file
+          break
+    return True
+
+  def _get_balance_sheet_history(self, position):
+    history = []
+    for stmt in self.module_data.get('balanceSheetHistory').get('balanceSheetStatements'):
```

Comment:
for these `.get()` calls and other ones in this change, can we provide a reasonable default in case the format of the data ever changes? So that it can fail gracefully instead of crash?

e.g. `.get('balanceSheetHistory', {})`. 

this should allow the chained `.get('balanceSheetStatements') to not-crash even if `balanceSheetHistory` key is not found.

Reply:
Added defaults.